editoast/src/views/timetable/stdcm.rs

```diff
@@ -642,3 +661,89 @@ enum SimulationTimeResult {
     SimulationTime { value: u64 },
     Error { error: Box<SimulationResponse> },
 }
+
+#[cfg(test)]
+mod tests {
+    use pretty_assertions::assert_eq;
+
+    use crate::views::rolling_stock::tests::fast_rolling_stock_form;
+
+    use super::*;
+
+    fn create_test_rolling_stock() -> RollingStockModel {
+        let rs_form = fast_rolling_stock_form("test_rolling_stock");
+        RollingStockModel {
+            id: 5656565,
+            name: rs_form.name,
+            loading_gauge: rs_form.loading_gauge,
+            supported_signaling_systems: rs_form.supported_signaling_systems,
+            version: 33,
+            base_power_class: rs_form.base_power_class,
+            comfort_acceleration: rs_form.comfort_acceleration,
+            inertia_coefficient: rs_form.inertia_coefficient,
+            startup_acceleration: rs_form.startup_acceleration,
+            startup_time: rs_form.startup_time,
+            effort_curves: rs_form.effort_curves,
+            electrical_power_startup_time: rs_form.electrical_power_startup_time,
+            raise_pantograph_time: rs_form.raise_pantograph_time,
+            energy_sources: rs_form.energy_sources,
+            gamma: rs_form.gamma,
+            locked: false,
+            metadata: rs_form.metadata,
+            power_restrictions: rs_form.power_restrictions,
+            railjson_version: "12".to_string(),
+            rolling_resistance: rs_form.rolling_resistance,
+            length: 140.0,       // m
+            mass: 150000.0,      // kg
+            max_speed: 110000.0, // m/s
```

Comment:
mm/s* ? 110 000 m/s seems a bit fast !

Reply:
it in m/s, https://github.com/OpenRailAssociation/osrd/blob/78160a257541d772fb4df95ff92b3218b463ea2d/python/osrd_schemas/osrd_schemas/rolling_stock.py#L206

The frontend rolling stock form also converts it and sends it in m/s.

Reply:
I suppose this comment is about the chosen speed being unrealistically fast 🚀